lib/spack/spack/config.py

```diff
@@ -1262,7 +1268,16 @@ def process_config_path(path):
             front = syaml.syaml_str(front)
             front.append = True
 
+        quote = "['\"]"
+        not_quote = "[^'\"]"
```

Comment:
(minor) can you move these definitions after the `result.append`?

Reply:
Sure, pushed.